packages/core/src/components/footer/footer.tsx

```diff
@@ -30,10 +33,12 @@ export class TdsFooter {
         <footer>
           {usesTopSlot && <slot name="top"></slot>}
           <div class="footer-main">
-            <div class="footer-main-top">
-              {usesStartSlot && <slot name="start"></slot>}
-              {usesEndSlot && <slot name="end"></slot>}
-            </div>
+            {!this.compactDesign && (
+              <div class="footer-main-top">
+                {usesStartSlot && <slot name="start"></slot>}
+                {usesEndSlot && <slot name="end"></slot>}
+              </div>
+            )}
```

Comment:
How tds-footer works today is that its size is dependent on fact if user is slotting or not slotting content inside of of different slots component has.

What do you think @HasanVatansever and @mJarsater if approach like this would be better?

Then we do not need additional prop, but pretty much we follow similar logic we have for top slot.
 If there is no slotted content, we will not render that part of a component at all?
 Please let me know what do you think of that approach. :) 


```suggestion
            { (usesStartSlot || usesEndSlot) && (
              <div class="footer-main-top">
                {usesStartSlot && <slot name="start"></slot>}
                {usesEndSlot && <slot name="end"></slot>}
              </div>
            )}
```

Reply:
Sounds good! Although, we might need to re-shuffle the styling some, right? The divide is set as border-top on the `footer-main-bottom`-class I belive.

Reply:
@theJohnnyMe!

Reply:
sounds nice 👍 @theJohnnyMe! .. would things be solved if suggestion  directly commited?,, just pushing commit!

Reply:
sounds nice 👍 @theJohnnyMe! .. would things be solved if suggestion  directly commited,, just pushing commit!

Reply:
@HasanVatansever yes if choose "commit suggestion" it will add the suggested code.